packages/logger/test/unit/basic-specs.js

```diff
@@ -422,4 +423,18 @@ describe('basic', async function () {
       o.includes('\u001b[7mtest message\u001b[0m').should.be.true;
     });
   });
+
+  describe('unleakString', function () {
+    it('should unleak a string', function () {
+      unleakString('yolo').should.eql('yolo');
```

Comment:
we should probably need to add an assertion that both objects have different references.
For that strings should be assigned to variables. And then we could compare references using the corresponding API from `support.node` set of helpers

Reply:
I agree that more tests are needed here, but the aim of this PR was to simply copy over the existing tests from the support package.